torch/csrc/api/src/mps.cpp

```diff
@@ -25,9 +21,22 @@ void manual_seed(uint64_t seed) {
 }
 
 void synchronize() {
-  TORCH_CHECK(is_available(), "No MPS devices are available");
   at::detail::getMPSHooks().deviceSynchronize();
 }
 
+void commit() {
+  at::detail::getMPSHooks().commitStream();
+}
+
+MTLCommandBuffer_t get_command_buffer() {
+  return static_cast<MTLCommandBuffer_t>(
+      at::detail::getMPSHooks().getCommandBuffer());
+}
+
+DispatchQueue_t get_dispatch_queue() {
+  return static_cast<DispatchQueue_t>(
+      at::detail::getMPSHooks().getDispatchQueue());
```

Comment:
Same as above, this is just a pass-thru
```suggestion
  return at::detail::getMPSHooks().getDispatchQueue();
```

Reply:
Same as above, it's a void* that needs to be cast to `dispatch_queue_t` before returning

Reply:
Updated the PR.